Fixes for 64 bits build issue:

 error: format '%llx' expects argument of type 'long long unsigned int', but argument 9 has type '__u64' {aka 'long unsigned int'} [-Werror=format=]

--- a/src/mesa/drivers/dri/i965/intel_batchbuffer.c
+++ b/src/mesa/drivers/dri/i965/intel_batchbuffer.c
@@ -67,7 +67,11 @@
       uint64_t flags = batch->validation_list[i].flags;
       assert(batch->validation_list[i].handle ==
              batch->exec_bos[i]->gem_handle);
+#if defined(__LP64__)
+      fprintf(stderr, "[%2d]: %2d %-14s %p %s%-7s @ 0x%016lx%s (%"PRIu64"B)\n",
+#else
       fprintf(stderr, "[%2d]: %2d %-14s %p %s%-7s @ 0x%016llx%s (%"PRIu64"B)\n",
+#endif
               i,
               batch->validation_list[i].handle,
               batch->exec_bos[i]->name,
@@ -721,7 +725,11 @@
 
       /* Update brw_bo::gtt_offset */
       if (batch->validation_list[i].offset != bo->gtt_offset) {
+#if defined(__LP64__)
+         DBG("BO %d migrated: 0x%" PRIx64 " -> 0x%lx\n",
+#else
          DBG("BO %d migrated: 0x%" PRIx64 " -> 0x%llx\n",
+#endif
              bo->gem_handle, bo->gtt_offset,
              batch->validation_list[i].offset);
          assert(!(bo->kflags & EXEC_OBJECT_PINNED));
